__tests__/getReviews.test.js

```diff
@@ -0,0 +1,57 @@
+const db = require("../db/connection.js");
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+
+describe("GET /api/reviews", () => {
+  it("should respond with a reviews array of review objects, each of which should have corresponding properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        for (let review of reviews) {
+          expect(review).toHaveProperty("review_id", expect.any(Number));
+          expect(review).toHaveProperty("owner", expect.any(String));
+          expect(review).toHaveProperty("title", expect.any(String));
+          expect(review).toHaveProperty("category", expect.any(String));
+          expect(review).toHaveProperty("review_img_url", expect.any(String));
+          expect(review).toHaveProperty("created_at", expect.any(String));
+          expect(review).toHaveProperty("votes", expect.any(Number));
+          expect(review).toHaveProperty("designer", expect.any(String));
+          if (review.id === 2 || review.id === 3) {
+            expect(review.comment_count).toBe(3);
```

Comment:
It's good that this is testing the specific number of reviews for ID 2 and 3 - however this means there could be no comment_count property on the other reviews and we wouldn't know, this test should be modified to expect a property of comment_count on every object with a value of any number. To check a specific review, you can do so by using it's index outside of the loop